Whoaverse/Whoaverse/Controllers/CommentController.cs

```diff
@@ -344,11 +344,11 @@ public ActionResult SubmitComment()
                 {
                     // notify comment author that his comment has been deleted by a moderator
                     MesssagingUtility.SendPrivateMessage(
-                        "Whoaverse",
+                        "Voat",
                         commentToDelete.Name,
                         "Your comment has been deleted by a moderator",
                         "Your [comment](/v/" + commentSubverse + "/comments/" + commentToDelete.MessageId + "/" + commentToDelete.Id + ") has been deleted by: " +
-                        "[" + User.Identity.Name + "](/u/" + User.Identity.Name + ")" + " on: " + DateTime.Now + "  " + Environment.NewLine +
+                        "[" + User.Identity.Name + "](/user/" + User.Identity.Name + ")" + " on: " + DateTime.Now + "  " + Environment.NewLine +
```

Comment:
Yes, the issue I linked still remains (it occurs in any place that accepts both markdown and the user mentions), the one I "fixed" is pretty much just a side effect of that one.